smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/commit/KafkaThrottledLatestProcessedCommit.java

```diff
@@ -41,37 +42,27 @@
  */
 public class KafkaThrottledLatestProcessedCommit implements KafkaCommitHandler {
 
-    private static final long THROTTLE_TIME_IN_MILLIS = 5_000L;
     private static final Map<String, Map<Integer, TopicPartition>> TOPIC_PARTITIONS_CACHE = new ConcurrentHashMap<>();
 
     private final Map<TopicPartition, OffsetStore> offsetStores = new HashMap<>();
 
     private final KafkaConsumer<?, ?> consumer;
     private final KafkaSource<?, ?> source;
-    private final int maxReceivedWithoutAckAllowed;
+    private final int unprocessedRecordMaxAge;
+    private final int autoCommitInterval;
 
     private volatile Context context;
-    private long nextCommitTime;
+
+    private Long timerId = null;
```

Comment:
while an object, a long with an initial value of -1 works well.

Reply:
Sure thing.